stacks/product-local/variables.tf

```diff
@@ -40,12 +29,27 @@ variable "enable_keycloak" {
   default = false
 }
 variable "builder_images_version" {
+  default = "v2.0.0"
```

Comment:
not a major issue but I think setting default versions in our stacks creates problems more often than it really helps anything

Reply:
gotcha, those were mostly there for testing so I removed this and the jenkins image version default